yellowbrick/cluster/elbow.py

```diff
@@ -247,6 +252,9 @@ def fit(self, X, y=None, **kwargs):
 
         self.k_scores_ = []
         self.k_timers_ = []
+        self.kneedle=None
+        self.knee_value=None
+        self.score=None
```

Comment:
This is great, thank you so much for storing these on the visualizer! I think users will be interested in directly accessing them. I've got two requests:

1. We should make these "learned" attributes and suffix them with an underscore, e.g. `self.knee_value_` and `self.score_`; also it is probably more informative if we name them `self.elbow_value_` and `self.elbow_score_` if that's ok -- see discussion above. 

2. We should document these in the docstring of the class. Also, it seems that `k_scores_`  and `k_timers_` are not documented, so would you mind also documenting them? See [icdm.py Line 123](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/cluster/icdm.py#L123) for an example of how to document the learned attributes. 

Finally (more on this later), these properties should only exist iff `self.locate_elbow==True`.

Reply:
@bbengfort Thanks so much for your kind words.

1. I was not aware of that naming convention of 'learned' attributes in yb but I found that in every visualizer class. I would change them as sugggested by you. Thanks.
2. I will document those attributes. They are important for other contributors too. Thanks.

And as suggested it makes sense for these properties to exist iff `self.locate_elbow==True`. I will make sure that happens.

Reply:
I appreciate it! The learned attributes are more a scikit-learn thing than a Yellowbrick thing. If you're interested in learning more, check out the [sklearn developer guide](https://scikit-learn.org/stable/developers/contributing.html#apis-of-scikit-learn-objects). I guess they call them "estimated attributes" there. 

A learned/estimated attribute is any data that is created when `fit()` is called - e.g. the `coef_` of a linear model. They only exist because data has been passed into the estimator. Because of this, the convention is to omit them from the class and only set them in `fit()` -- this also is what we use to check if an estimator is fitted or not. 

Because you're making the `KneeLocator` a local variable, not storing it on the visualizer, it is not a learned attribute. However, `elbow_value_`, `elbow_score_`, `k_scores_`, and `k_timers_` are all learned attributes because they do not/cannot exist until `fit()`.